app/js/constants/droits.js

```diff
@@ -176,5 +176,21 @@ angular.module('ddsCommon').constant('droitsDescription', {
         'imgSrc':'logo_mairie_paris.png',
         'link':'http://www.paris.fr/aides',
         'isBaseRessourcesYearMoins2':false
+    },
+    'aah': {
+        'id': 'aah',
+        'isMontantAnnuel': false,
+        'label': 'Allocation aux adultes handicapés',
+        'shortLabel':'AAH',
+        'imgSrc':'logo_caf.png',
+        'description':'L’AAH est une aide financière qui permet d’assurer un revenu minimum aux adultes handicapés. ',
+        'isBaseRessourcesYearMoins2':true,
+        'uncomputability': {
+            'intervention_CDAPH_necessaire': {
+                'reason': 'le montant de l’AAH dépend du taux d’incapacité déterminé par la Commission des droits et de l’autonomie des personnes handicapées (CDAPH) après le dépôt de votre demande',
+                'solution': 'Vous pouvez contacter <a href="http://informations.handicap.fr/carte-france-mdph.php"> la MDPH la plus proche</a> pour faire établir ce taux.'
```

Comment:
No spaces within HTML tags.

> They behave differently based on the type of tag, and styling may apply on blank spaces, so to avoid surprises, always use spaces on the outer parts of the tags.